cmd/app/main.go

```diff
@@ -23,7 +23,7 @@ func main() {
 			die(err)
 		}
 
-		manifest, err = models.LoadManifest(string(data))
+		manifest, err = models.LoadManifest(string(data), nil)
```

Comment:
@ddollar I'm really confused about where/how/why this is used. Can you shed some light? I'm also not sure if we can get away with nil here...